source/adapter.js

```diff
@@ -971,10 +972,8 @@ if ( navigator.mozGetUserMedia ||
       if (AdapterJS.WebRTCPlugin.plugin.PEER_CONNECTION_VERSION &&
           AdapterJS.WebRTCPlugin.plugin.PEER_CONNECTION_VERSION > 1) {
         // RTCPeerConnection prototype from the new spec
-        if (iceServers) {
-          servers.iceServers = iceServers;
-        }
-        return AdapterJS.WebRTCPlugin.plugin.PeerConnection(servers);
+        var s = { iceServers: iceServers };
```

Comment:
Hmm, if I'm not sure if this is a good choice since if I pass `{ iceTransportsPolicy: 'relay', iceServers: theServers }` it will only enforce using `iceServers` in the `RTCConfiguration`.